BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -522,21 +523,27 @@ def sanitize_param(input)
       # Make sure the file ID is valid, accessible, not already used and
       # of the correct type.
       when :file
-        unless (Integer(value) rescue nil)
+        if !(Integer(value) rescue nil) && !single_file_input?
```

Comment:
I don't understand why the fact we have a single file input is important when validating that a file's value is an integer? It shoudl always be true, no?